CMakeLists.txt

```diff
@@ -4,6 +4,12 @@ project(ftxui
   VERSION 0.1
 )
 
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+add_compile_options("$<$<C_COMPILER_ID:MSVC>:/utf-8>")
```

Comment:
What about using target_compile_option(<target> PRIVATE "your option")
This avoid affecting other targets the user of the library might have.

Reply:
Will fix, this was a remainder from my first try.